run_page/tcx_to_strava_sync.py

```diff
@@ -57,19 +60,25 @@ def get_to_generate_files(last_time):
     to_upload_time_list, to_upload_dict = get_to_generate_files(last_time)
     index = 1
     for i in to_upload_time_list:
-        tcx_file = to_upload_dict.get(i)
+        tcx_file = to_upload_dict.get(i)["file"]
+        type = to_upload_dict.get(i)["type"].lower()
+        if type not in STRAVA_GARMIN_TYPE_DICT.values():
+            continue
+        strava_type = list(STRAVA_GARMIN_TYPE_DICT.keys())[
```

Comment:
How about storing a reverted dict first:
```
GARMIN_STRAVA_TYPE_DICT = {value: key for key, value in STRAVA_GARMIN_TYPE_DICT.items()}
```
and this 3 lines can be replace by 
```
strava_type = GARMIN_STRAVA_TYPE_DICT[type]
```

Reply:
OK，I'll fix it.

Reply:
[89ed833](https://github.com/yihong0618/running_page/pull/666/commits/89ed83355cfc765abbf2e6fa819c59e6af231c95)
Fixed here.